pinot/net.py

```diff
@@ -18,22 +18,78 @@ class Net(torch.nn.Module):
     learning, puts into a predicted distribution and calculates the
     corresponding divergence.
 
+
+    Attributes
+    ----------
+    representation: a `pinot.representation` module
+        the model that translates graphs to latent representations
+    output_regression: a `torch.nn.Moudle` or None,
+        if None, this will be set as a simple `Linear` layer that inputs
+        the latent dimension and output the number of parameters for
+        `self.distribution_class`
+    distribution_class: a `torch.distribution` object,
+        default=`torch.distributions.normal.Normal`
+        the class of distribution the model outputs
+    noise_model: either a string (
+        one of 
+            'homoschedastic',
+            'heteroschedastic',
+            'homoschedastic-fixed') 
+        or a function that transforms a set of parameters.
+
+
     """
 
     def __init__(
         self,
         representation,
-        parameterization,
+        output_regression=None,
         distribution_class=torch.distributions.normal.Normal,
```

Comment:
same thing as above... I would imagine all of this can be absorbed into noise model.